controller/src/main/java/org/jboss/as/controller/ListAttributeDefinition.java

```diff
@@ -184,6 +193,10 @@ protected abstract void addOperationParameterValueTypeDescription(final ModelNod
                                                                       final ResourceDescriptionResolver resolver,
                                                                       final Locale locale, final ResourceBundle bundle);
 
+    protected abstract void addOperationReplyValueTypeDescription(final ModelNode node, final String operationName,
+                                                                      final ResourceDescriptionResolver resolver,
+                                                                      final Locale locale, final ResourceBundle bundle);
+
```

Comment:
In the full WildFly PR I call *getOperationReplyValueTypeDescription()* instead of *getOperationParameterValueTypeDescription()* . While getOperationReplyValueTypeDescription() falls back to getOperationParameterValueTypeDescription() if no specific reply description is found this is a difference.